### PR TITLE
Add more nice castor command

### PR DESCRIPTION
**Pull request by lyrixx**

### CODE REVIEW

File: castor.php [hunk 1]
```diff
@@ -105,3 +107,20 @@ function migrate(): void
...
+    // docker_compose_run('bin/console doctrine:fixture:load -n');
+}
+
+
+#[AsTask(description: 'Connect to the PostgreSQL database', name: 'db:client', aliases: ['postgres'])]
+function postgres_client(): void
+{
+    io()->title('Connecting to the PostgreSQL database');
+
+    docker_compose(['exec', 'postgres', 'psql', '-U', 'app', 'training'], context()->toInteractive());
```
Comment (pyrech) [inline castor.php]:
```suggestion
    docker_compose(['exec', 'postgres', 'psql', '-U', 'app', 'app'], context()->toInteractive());
```

File: castor.php [hunk 2]
```diff
@@ -105,3 +107,20 @@ function migrate(): void
...
+#[AsTask(description: 'Loads fixtures', namespace: 'app:db', aliases: ['fixture'])]
+function fixtures(): void
+{
+    // io()->title('Loads fixtures');
+
+    // docker_compose_run('bin/console doctrine:fixture:load -n');
+}
+
+
+#[AsTask(description: 'Connect to the PostgreSQL database', name: 'db:client', aliases: ['postgres'])]
```
Comment (pyrech) [inline castor.php:120]:
```suggestion
#[AsTask(description: 'Connect to the PostgreSQL database', name: 'db:client', aliases: ['postgres', 'pg'])]
```

Team lazy :raised_back_of_hand: :grin:

**Review by pyrech — COMMENTED** [review]
LGTM

File: castor.php (hunk 2 above)
Comment (lyrixx) [reply castor.php]: Done ✅

**Review by pyrech — APPROVED** [review]
Awesome thanks

File: castor.php [hunk 3]
```diff
@@ -105,3 +107,19 @@ function migrate(): void
...
+
+#[AsTask(description: 'Loads fixtures', namespace: 'app:db', aliases: ['fixture'])]
+function fixtures(): void
+{
+    // io()->title('Loads fixtures');
+
+    // docker_compose_run('bin/console doctrine:fixture:load -n');
+}
+
+#[AsTask(description: 'Connect to the PostgreSQL database', name: 'db:client', aliases: ['postgres', 'pg'])]
```
Comment (JorickPepin) [inline castor.php:119]: the namespace doesn't seem to be ISO compared to other tasks

Comment (lyrixx) [reply castor.php:119]: What would you suggest?

Comment (JorickPepin) [reply castor.php:119]: I personally prefer to have just `db:` rather than `app:db:`

Comment (lyrixx) [reply castor.php:119]: Perfect, this is what I did initially 😅 🤗

**Pull request merged**